docs/en/sql-reference/functions/uuid-functions.md

```diff
@@ -129,6 +129,60 @@ SELECT generateUUIDv7(1), generateUUIDv7(2);
 └──────────────────────────────────────┴──────────────────────────────────────┘
 ```
 
+## dateTimeToUUIDv7 {#datetimetouuidv7}
+
+Converts a [DateTime](../data-types/datetime.md) value to the first [UUIDv7](https://en.wikipedia.org/wiki/UUID#Version_7) at the given time.
```

Comment:
So the counter and random parts of the returned UUIDv7 are random, right? How can the function then return the "first" UUIDv7 at the given time?